src/tools/imgtool/main.cpp

```diff
@@ -106,15 +111,15 @@ static int parse_options(int argc, char *argv[], int minunnamed, int maxunnamed,
 	return lastunnamed;
 
 filternotfound:
-	fprintf(stderr, "%s: Unknown filter type\n", value);
+	std::wcerr << wstring_from_utf8(value) << L": Unknown filter type" << std::endl;
 	return -1;
 
 optionalreadyspecified:
-	fprintf(stderr, "Cannot specify multiple %ss\n", name);
+	std::wcerr << L"Cannot specify multiple " << wstring_from_utf8(name) << L"s" << std::endl;
```

Comment:
As well as needing to use a format-style form to ever be localisable, assuming you can turn something into a plural is very bad.  It needs to be reworded to not make the assumption, or whatever yields the name needs to be changed to be able to produce a plural on demand.